### ISSUE TITLE
Accessibility labels missing from App Guides screen

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#1184]
### Description of problem

Accessibility labels missing from "App Guides".

### Expected behavior

Each radio toggle in "App Guides" should have an accessibility label associated. Probably needs a capital "G" in the title also.

### Steps to reproduce

1. On iOS enable Voice Control;
2. Go to App Guides from Settings.
3. Say "Show names".

### Screenshots and/or log output

![Screenshot 2023-06-13 at 5.21.39 PM.png](https://images.zenhubusercontent.com/5c2fa1b7ce4f66381cd8d02d/b877f1d6-81b1-4406-bab4-782ef8a30e33)

### Environment

- [ ] Occurs on Android
- [x] Occurs on iOS

Build #:

Android Device Model: 

iOS Device Model: 

iPhone 14 Pro

### Severity
- [ ] High
- [ ] Medium
- [x] Low
